cypress/integration/shared.js

```diff
@@ -114,17 +114,18 @@ export const generateIso = (sshPubKey) => {
   // click to download the discovery iso
   cy.get('#button-download-discovery-iso').click();
   // see that the modal popped up
-  cy.get('h1#pf-modal-part-8').should('be.visible');
+  cy.get('#pf-modal-part-6').should('be.visible');
   // feed in the public ssh key
-  pasteText(cy, '#form-input-sshPublicKey-discovery-field', sshPubKey);
+  cy.get('#form-input-sshPublicKey-discovery-field').type(sshPubKey);
   let aborted = false;
   cy.server({
     onAnyAbort: (...args) => {
       aborted = true;
       console.log('-- onAnyAbort: ', ...args);
     },
   });
-  cy.get('.pf-c-modal-box__footer > .pf-m-primary').contains('Get Discovery ISO');
+  cy.get('.pf-c-modal-box__footer > .pf-m-primary').should('be.visible');
```

Comment:
having both should('be.visible') and contains('Generate Discovery ISO') seems redundant; wdyt?

Reply:
be.visible means that the button is clickable where the contains only check its text, correct me if im wrong

Reply:
If the element is not visible the contains('Generate Discovery ISO') will fail. But failing on should('be.visible') gives a better indication of what failed.